nfpm.go

```diff
@@ -106,23 +109,26 @@ func (c *Config) Validate() error {
 
 // Info contains information about a single package
 type Info struct {
-	Overridables `yaml:",inline"`
-	Name         string `yaml:"name,omitempty"`
-	Arch         string `yaml:"arch,omitempty"`
-	Platform     string `yaml:"platform,omitempty"`
-	Epoch        string `yaml:"epoch,omitempty"`
-	Version      string `yaml:"version,omitempty"`
-	Release      string `yaml:"release,omitempty"`
-	Prerelease   string `yaml:"prerelease,omitempty"`
-	Section      string `yaml:"section,omitempty"`
-	Priority     string `yaml:"priority,omitempty"`
-	Maintainer   string `yaml:"maintainer,omitempty"`
-	Description  string `yaml:"description,omitempty"`
-	Vendor       string `yaml:"vendor,omitempty"`
-	Homepage     string `yaml:"homepage,omitempty"`
-	License      string `yaml:"license,omitempty"`
-	Bindir       string `yaml:"bindir,omitempty"`
-	Target       string `yaml:"-"`
+	Overridables     `yaml:",inline"`
+	Name             string `yaml:"name,omitempty"`
+	Arch             string `yaml:"arch,omitempty"`
+	Platform         string `yaml:"platform,omitempty"`
+	Epoch            string `yaml:"epoch,omitempty"`
+	Version          string `yaml:"version,omitempty"`
+	Release          string `yaml:"release,omitempty"`
+	Prerelease       string `yaml:"prerelease,omitempty"`
+	Section          string `yaml:"section,omitempty"`
+	Priority         string `yaml:"priority,omitempty"`
+	Maintainer       string `yaml:"maintainer,omitempty"`
+	Description      string `yaml:"description,omitempty"`
+	Vendor           string `yaml:"vendor,omitempty"`
+	Homepage         string `yaml:"homepage,omitempty"`
+	License          string `yaml:"license,omitempty"`
+	Bindir           string `yaml:"bindir,omitempty"`
+	PrivateKeyBase64 string `yaml:"privatekeybase64,omitempty"`
```

Comment:
for those `apk`-only properties, maybe we could create a `Apk` struct?

check for instance `RPM` and `Deb` bellow in this same file.